tests/e2e/default-mobile/wdio.conf.js

```diff
@@ -6,7 +6,10 @@ chai.use(require('chai-exclude'));
 // Override specific properties from wdio base config
 exports.config = Object.assign(wdioBaseConfig.config, {
   specs: [
-    '**/*.wdio-spec.js'
+    '**/*.wdio-spec.js',
+    [
+      '../default/login/login-logout.wdio-spec.js',
+    ],
```

Comment:
@ngaruko in another [PR](https://github.com/medic/cht-core/pull/8255) @latin-panda suggested we could group tests by concerns using brackets to avoid mixing particular logic in desktop tests. 
[This](https://github.com/medic/cht-core/pull/8255#discussion_r1197612353) is the comment.

Reply:
About the file naming - Instead of naming a file `navigation.mobile.wdio.spec.js` with `*.mobile.wdio.spec.js` suffix, better place the file in the mobile e2e folder `/default-mobile/**/navigation.wdio-spec.js`. 

Also, after migrating all protractor, there can be another session with the team to analyze and design a test structure that better fits mobile.  Since we still have few tests for mobile.

Reply:
Thanks @ngaruko @latin-panda. I'm going to place the file in the mobile e2e folder. I'm going to do that also for the common spec. @latin-panda do you think I have to add this specs into the `tests/e2e/default/wdio.conf.js` or it is a better idea that the test run only on mobile?

Reply:
Today I had a closer look at these specs, so the tests from: `tests/mobile/login/login.specs.js` (Protractor) are now in `tests/e2e/default/login/login-logout.wdio-spec.js` (desktop wdio), which I think it's okay also to verify these scenarios are working in desktop. 

In my opinion, you can leave it like this. We're not losing track of what's tested in mobile because it's already mapped in  `tests/e2e/default-mobile/wdio.conf.js` and we have the commits history. 

For tests that are mobile-only (like FAB in list views), they should be located in `/default-mobile/`.

When the protractor migration is complete, then we can evaluate the mobile testing and make a plan to structure it better 🙂

Reply:
Perfect! Thank you @latin-panda. 
What do you think about that decision @ngaruko ?